io/router/transfer.go

```diff
@@ -0,0 +1,43 @@
+package router
+
+import (
+	"gioui.org/io/transfer"
+	"strings"
+)
+
+type transferQueue struct {
+	handlers      []transfer.SchemeOp
+	firstURLEvent transfer.URLEvent
```

Comment:
After further thought, I think `firstURLEvent` and the delaying of it belongs in app.Window. I don't think the router package should know about the quirk that the first event is delayed.